aladdin-container.sh

```diff
@@ -102,7 +102,7 @@ function environment_init() {
         _handle_authentication_config
 
         if $INIT; then
-            $PY_MAIN create-namespace $NAMESPACE || true
+            $ALADDIN_DIR/aladdin/bash/container/create-namespace/create-namespace $NAMESPACE || true
```

Comment:
`create-namespace` is a bash plugin, but we are already inside the container, so we need to call the script directly (can't use `PY_MAIN`)